src/Components/Header.tsx

```diff
@@ -0,0 +1,21 @@
+import { Link } from 'react-router-dom';
+
+export const Header: React.FC = () => {
+  return (
+    <header>
+      <nav className="navbar">
+        <Link to="/" className="navbar-item">
+          Home Page
+        </Link>
+
+        <Link to="/home" className="navbar-item">
+          Test redirection from HOME
+        </Link>
```

Comment:
I don't think that you need this link. [Checklist 6](https://github.com/mate-academy/react_people-table-basics)